couch/tests/compose/compose_v1.yaml

```diff
@@ -6,3 +6,4 @@ services:
     image: couchdb:${COUCH_VERSION}
     ports:
       - ${COUCH_PORT}:5984
+    platform: linux/x86_64
```

Comment:
Could you check why this was necessary?

Reply:
It's necessary because I'm on an Apple Silicon Mac, and [`couchdb:1.6`](https://hub.docker.com/layers/library/couchdb/1.6/images/sha256-827c620e200476d045ca067450590f43595028857073f06af412ec379d89b872?context=explore) and [`couchdb:2.1`](https://hub.docker.com/layers/library/couchdb/2.1/images/sha256-0db0c4109b9256db4371ae20dc048b6ad432d19b3bbbea414f637cb79f7211b0?context=explore) don't have an ARM builds. [`couchdb:3.1`](https://hub.docker.com/layers/library/couchdb/3.1/images/sha256-dd4981369483a912a4f0da3e9323e76e9a099aded11308a0cc2d163abf34c10d?context=explore) does.

Though now I'm confused that `compose_v2.yaml` doesn't need the same change, so perhaps I'm wrong on the root cause here.

Reply:
Ah, it's because we use [`couchdb:2.3`](https://hub.docker.com/layers/library/couchdb/2.3/images/sha256-7dda1442f1ed61b3177c8d6b48e0b83e999150fb5ea32fd16d72aa0c25b59ed1?context=explore) in the tests, not 2.1. That makes sense, 2.3 does have an ARM build.